helm/fluent-logshipping-app/templates/fluentbit/configmap.yaml

```diff
@@ -86,17 +128,48 @@ data:
         Match           *
         Host            ${FLUENTD_HOST}
         Port            ${FLUENTD_PORT}
+        Retry_Limit     false              # Retry
+
+  outputs-azure-loganalytics.conf: |
```

Comment:
Where do you have the `include` directive for this one?

Reply:
I actually do not, as this is wip to see if removing fluentd is worth it or not :( Let me remove it for this PR